File: BBDDLib/Models/InvestmentProductsPrices.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace BBDDLib.Models
{
    public class InvestmentProductsPrices
    {
        [Key]
        public virtual int id { set; get; }
        public virtual DateTime? date { set; get; }
        public virtual int? investmentProductsid { set; get; }
        public virtual InvestmentProducts? investmentProducts { set; get; }
        public virtual Decimal? prices { set; get; }
    }
}

File: WebApp/app/Models/InvestmentProductsPrices.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class InvestmentProductsPrices extends Model
{
    use HasFactory;

    protected $table = 'investment_products_prices';

    protected $fillable = ['date', 'investmentProductsid', 'prices'];
}
